src/utils/components/withMatchMedia.test.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import TestUtils from 'react-addons-test-utils';
+
+import {MEDIA_QUERIES} from '../designConstants';
+import * as MM from './withMatchMedia';
+
+const allBreakpoints = Object.keys(MEDIA_QUERIES);
+
+/**
+ * @class TestComponent
+ */
+class TestComponent extends React.Component {
+	render() {
+		return <h1>Hello world</h1>;
+	}
+}
+
+/**
+ * @param {Array} breakpoints - breakpoint names to pass to withMatchMedia
+ */
+function renderWrappedComponent(breakpoints) {
+	const renderer = TestUtils.createRenderer();
+	const TestComponentWithMatchMedia = MM.withMatchMedia(
+		TestComponent,
+		breakpoints
+	);
+	renderer.render(<TestComponentWithMatchMedia />);
+	return renderer.getRenderOutput();
```

Comment:
instead of shallow rendering, try using:
```jsx
TestUtils.renderIntoDocument(<TestComponentWithMatchMedia />)
```

I'll have to look a bit more into how to use Shallow Rendering to get the root level component